config/application.rb

```diff
@@ -15,7 +15,13 @@
 module Pupilfirst
   class Application < Rails::Application
     # Initialize configuration defaults for originally generated Rails version.
-    config.load_defaults 6.1
+    config.load_defaults 7.0
+
+    config.active_support.cache_format_version = 7.0
```

Comment:
Only `load_defaults 7.0` needs to be mentioned. Setting `cache_format_version = 7.0` does not do anything extra.

This is documented here: https://edgeguides.rubyonrails.org/upgrading_ruby_on_rails.html#new-activesupport-cache-serialization-format